docs/api-guide/throttling.md

```diff
@@ -23,6 +23,9 @@ As with permissions and authentication, throttling in REST framework is always d
 Before running the main body of the view each throttle in the list is checked.
 If any throttle check fails an `exceptions.Throttled` exception will be raised, and the main body of the view will not run.
 
+**Note:** Prior to version 3.10, throttle checking stopped on the first failure encountered.
```

Comment:
This is worded in a way to describe the implementation which makes it hard to understand the change in behavior. How about something to explain how the effect of throttling will be different before and after 3.10? Here is a suggestion:

Prior to version 3.10, throttling was less effective because throttled requests did not count towards quotas. This was fixed in 3.10 by checking all requests, including throttled requests.

Reply:
Agreed, thanks @kumar303. Phrasing could be a bit more behavior based rather than implementation based here.